client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/address-step/unverified.js

```diff
@@ -112,12 +112,12 @@ const UnverifiedAddress = ( {
 						) }
 					</p>
 					{ uspsUrl && (
-						<ExternalLink icon={ true } href={ uspsUrl } target="_blank">
-							{ translate( 'Verify with USPS' ) }
+						<ExternalLink href={ uspsUrl } target="_blank">
+							{ translate( 'Verify with USPS ' ) }
```

Comment:
Icon is included by default on WordPress component.

Reply:
It looks like also `target="_blank"` is not necessary: https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/external-link/index.js#L34

Reply:
Thanks! Removed.